contracts/coordinator/src/msg.rs

```diff
@@ -15,20 +15,14 @@ pub enum ExecuteMsg {
         chain_name: ChainName,
         new_prover_addr: Addr,
     },
-    SetActiveVerifiers {
-        next_verifier_set: VerifierSet,
-    },
-    SetNextVerifiers {
-        next_verifier_set: VerifierSet,
+    UpdateVerifierUnionSet {
+        union_set: HashSet<Addr>,
```

Comment:
```suggestion
    SetVerifiers {
        verifiers: HashSet<Addr>,
```

The fact that the set is a union of two sets is an implementation detail that may not always be true